### PR TITLE
Add site tours page

### PR DESCRIPTION
**Pull request by rwwagner90**

### CODE REVIEW

File: pages/app-tours.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    <HeroBlock class="app-tours">
+      <template v-slot:content>
+        <h1
+          class="heading"
+          itemprop="name"
+        >
+          App Tours with Shepherd.js
+        </h1>
+        <p>
+          We leverage <a href="https://shepherdjs.dev/">Shepherd.js</a> to create step-by-step app tours and feature
```
Comment (chuckcarpenter) [inline pages/app-tours.vue]:
```suggestion
          We leverage <a href="https://shepherdjs.dev/">Shepherd.js</a> to create step-by-step application tours and feature
```

Comment (chuckcarpenter) [reply pages/app-tours.vue]: just thinking we should add different wording combinations

File: pages/app-tours.vue [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        <h1
+          class="heading"
+          itemprop="name"
+        >
+          App Tours with Shepherd.js
+        </h1>
+        <p>
+          We leverage <a href="https://shepherdjs.dev/">Shepherd.js</a> to create step-by-step application tours and feature
+          introductions to help new users get up to speed quickly with your web app.
+        </p>
```
Comment (chuckcarpenter) [inline pages/app-tours.vue]:
```suggestion
<p>          
Welcome your users, introduce new features, or just lead them through an important process by adding tours. 
</p>
<p>
We've leveraged the flexibility of these tours to lead users through a number of events. Upon the first time logging into an application, tours can be used to validate and ensure a smooth setup process.
</p>
<p>
Your existing visitors can be shown exciting new features and how to maximize their ability to complete important tasks to ensure long term effectiveness for investments you've made to improve your software.
</p>
<p>
<a href="https://shepherdjs.dev/">Shepherd.js</a> makes this easy to add, customize, and integrate into your website or application. 
</p>
```

File: pages/app-tours.vue [hunk 3]
```diff
@@ -10,7 +10,7 @@
         </h1>
         <p>
           We leverage <a href="https://shepherdjs.dev/">Shepherd.js</a> to create step-by-step application tours and feature
-          introductions to help new users get up to speed quickly with your web app.
+          introductions to help new users get up to speed quickly with your web application.
```
Comment (rwwagner90) [inline pages/app-tours.vue]: @chuckcarpenter I think we should leave this as `app` so we say both application and app, rather than just application.

**Pull request merged**